conf/messages.en

```diff
@@ -478,3 +463,40 @@ paymentsIntoPensions.checkYourAnswersLabel = paymentsIntoPensions
 paymentsIntoPensions.error.required = Select paymentsIntoPensions
 paymentsIntoPensions.agent.error.required = Select paymentsIntoPensions
 paymentsIntoPensions.change.hidden = PaymentsIntoPensions
+
+###################################  UK interest ###########################################################################################
```

Comment:
I know the welsh content isnt ready yet but could we add the english messages into the welsh content just as placeholders?